scripts/lint-lists.py

```diff
@@ -109,6 +121,8 @@ def main(source='OONI', notes='', legacy=False, fix_duplicates=False):
         )
     header = ['url', 'category_code', 'category_description',
               'date_added', 'source', 'notes']
+    # preload the global list to check against looking for dupes
+    check_against_global = load_global_list(glob(os.path.join(lists_path, "global.csv"))[0])
```

Comment:
If you make the above change to use a `set`, I would suggest renaming this to `global_urls_bag` for consistency with the `urls_bag` below.

Reply:
Also if you have passing the path the list (which is precise) you don't need to use `glob` in front of this, but you can just say: `load_global_list(os.path.join(lists_path, "global.csv"))`